setup.py

```diff
@@ -46,7 +46,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 
-VERSION = '1.4.3'
+VERSION = '1.4.4'
```

Comment:
Please rebase your branch and update the version to 1.4.5. The latest release is 1.4.4

Reply:
Yes, sorry about that, I missed your earlier comment about this :flushed: